app/main/helpers/search_save_helpers.py

```diff
@@ -38,3 +38,27 @@ def __init__(self, search_api_url, frameworks_by_slug, include_markup=False):
             filters.values(),
             lots_by_slug
         )
+
+
+def get_saved_search_temporary_message_status(project, framework, following_framework):
+    if not project['lockedAt']:
+        if following_framework['status'] in ['coming', 'open', 'pending', 'standstill']:
+            return 'not_locked_pre_live'
```

Comment:
These return value strings should probably be declared once as constant globals so that we aren't just hand writing them in various places.

We'll need to add them to the Jinja templating environment but that shouldn't be too difficult: http://jinja.pocoo.org/docs/2.10/api/#global-namespace

Reply:
The more I think about this the odder I find it. Totally get making the strings in constants, but at that point the string itself loses any real meaning. I'm only using them as way of identifying something and not for their actual content.
And once I've added them to the Jinja environment I'm going to have a similar issue, that I'm going to be handwriting the environment variables name, instead of just the string I'm using, so the same risk but with more faff. Maybe I'm missing something. What do you think?